src/docs/product/performance/query-insights.md

```diff
@@ -0,0 +1,131 @@
+
+Starting with the [**Queries** page](#queries-page), you get a high-level overview of how your queries are doing. From there, you can drill into a specific query's [**Query Summary** page](#query-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific endpoint.
+
+The gif below demonstrates how to use Query Insights.
+
+<div style="position: relative; padding-bottom: calc(49.96436208125446% + 41px); height: 0; width: 100%"><iframe src="https://demo.arcade.software/D5eQ8zzun3iYBF4Ha3mh?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;" title="Performance - Query Insights - Walkthrough"></iframe></div>
+
+</br>
+
+## Prerequisites and Limitations
+
+The queries widget and pages are only available for backend projects with performance monitoring enabled.
+
+<Note>
+
+The **Queries** page collects queries from your application's endpoints. Queries that run in async tasks are not shown. If a query runs in an endpoint _and_ a task, the metrics will reflect its performance within endpoints only.
+
+</Note>
+
+### Recommended SDK Versions
+
+Query Insights work best with up-to-date SDK versions. The following minimum versions are recommended:
+
+- [Python SDK v1.29.2](https://github.com/getsentry/sentry-python/releases/tag/1.29.2)
+- [JavaScript SDKs v7.63.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.63.0)
+- [Laravel SDK v3.8.0](https://github.com/getsentry/sentry-laravel/releases/tag/3.8.0)
+- [Cocoa SDK v8.11.0](https://github.com/getsentry/sentry-cocoa/releases/tag/8.11.0)
+- [Java SDK v6.29.0](https://github.com/getsentry/sentry-java/releases/tag/6.29.0)
+- [Ruby SDK v5.11.0](https://github.com/getsentry/sentry-ruby/releases/tag/5.11.0)
+- [.NET SDK v3.39.0](https://github.com/getsentry/sentry-dotnet/releases/tag/3.39.0)
+- [Symfony SDK v4.11.0](https://github.com/getsentry/sentry-symfony/releases/tag/4.11.0)
+
+### Span Eligibility
+
+The data shown is pulled from `db` and `db.sql` spans.
+
+Sentry tries to extract metrics for all SQL-like dialects. NoSQL databases like MongoDB, graph databases like Neo4j, and other non-SQL database systems are not eligible for this feature.
+
+If you are using <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation</PlatformLink>, Query Insights should work without any configuration. If you've manually instrumented Sentry, you'll need to make sure that your spans conform to our standards for the best experience:
+
+- The span `op` field is set to an [eligible value](https://develop.sentry.dev/sdk/performance/span-operations/#database).
+- The span's description contains the full SQL query.
+- The `db.system` span data value is set to the correct identifier (e.g., `"postgresql"` or `"mysql"`).
```

Comment:
This matches https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#notes-and-well-known-identifiers-for-dbsystem - do we want to link there?